.github/workflows/build.yml

```diff
@@ -5,30 +5,13 @@ on: [push]
 jobs:
   build:
 
-    name: Python ${{ matrix.python-version }}, Django ${{ matrix.django-version }} (${{ matrix.variant }})
+    name: Python ${{ matrix.python-version }}
     runs-on: ubuntu-latest
-    continue-on-error: ${{ matrix.experimental == 'true' }}
     strategy:
       matrix:
-        python-version: ['3.5', '3.6', '3.7', '3.8']
-        django-version: ['2.2', '3.0', '3.1', master]
-        variant: [normal, yubikey, custom_user]
-        experimental: ['true', 'false']
```

Comment:
This changes the build matrix to only have the python versions, and using tox to build the remaining matrix within that version. I wrote it that way to have a nice overview inside Github to inspect the matrix output. On the other hand, the UI assumes smaller matrices and having such a long list can be harder to work with. What's your take on this?

See also https://github.com/Bouke/django-two-factor-auth/pull/430 that enables this workflow for PRs.

Reply:
Personally I find the duplicated build matrix to be a bit of nightmare for project maintenance, which is why I changed it.